test/object-store/migrations.cpp

```diff
@@ -191,6 +191,183 @@ auto create_objects(Table& table, size_t count)
 }
 } // anonymous namespace
 
+TEST_CASE("migration: Additive mode returns OS schema - Automatic migration") {
+
+    SECTION("Check OS schema returned in additive mode") {
```

Comment:
All of the added tests are checking differences in entire tables, what about checking behaviour of missing properties as well?

Reply:
Yep, I did not add more, because I extended the previous test where the removed properties were expected to be removed, but right now they are actually appended to the list of `persisting_properities` 
`TEST_CASE("migration: AdditiveDiscovered")`